README.md

```diff
@@ -113,6 +113,81 @@ extension SpeechlyManager: SpeechClientDelegate {
 
 Check out the [ios-repo-filtering](https://github.com/speechly/ios-repo-filtering) repository for a demo app built using this client.
 
+### User interface components
+
+The client library also includes a couple of ready-made UI components which can be used together with `SpeechClient`.
+
+`SpeechButton` starts and stops voice recognition automatically when pressed and released, using build-in icons and visual effects which you can replace with your own if needed.
+
+`SpeechTranscriptView` visualizes the transcripts received in the `speechlyClientDidUpdateSegment` callback, automatically highlighting recognized entities.
+
+These can be used, for example, in the following way:
+
+```swift
+import UIKit
+import SnapKit
+import Speechly
+
+class ViewController: UIViewController, SpeechClientDelegate, SpeechButtonDelegate {
+    
+    private let client: SpeechClient
+    
+    private lazy var speechButton = SpeechButton(delegate: self)
+    
+    private let transcriptView = SpeechTranscriptView()
+    
+    init() {
+        client = try! SpeechClient(
+            appId: UUID(uuidString: "your-speechly-app-id")!,
+            language: .enUS
```

Comment:
`language` is not needed here anymore.

Reply:
Hmm. Seems to me both `SpeechClient` initializers would still require it.

Reply:
There is no language in at least https://github.com/speechly/ios-client/blob/master/Sources/Speechly/Speech/SpeechClient.swift#L112, do you have a pointer to the place where it is used?

Reply:
Argh, sorry, I was looking at the code checked out under my demo app, which was outdated!